misc/ballerina-bindgen/src/main/java/org/ballerinalang/bindgen/model/JClass.java

```diff
@@ -111,52 +108,66 @@ public JClass(Class c) {
     }
 
     private void populateConstructors(Constructor[] constructors) {
-
         int i = 1;
         for (Constructor constructor : constructors) {
             JConstructor jConstructor = new JConstructor(constructor);
-            jConstructor.setConstructorName("new" + this.shortClassName + i);
-            this.constructorList.add(jConstructor);
+            jConstructor.setConstructorName("new" + shortClassName + i);
+            constructorList.add(jConstructor);
             i++;
         }
     }
 
     private void populateInitFunctions() {
-
         int j = 1;
-        for (JConstructor constructor : this.constructorList) {
+        for (JConstructor constructor : constructorList) {
             JConstructor newCons = null;
             try {
                 newCons = (JConstructor) constructor.clone();
-            } catch (CloneNotSupportedException e) {
-                errStream.println(e);
+            } catch (CloneNotSupportedException ignore) {
+
             }
```

Comment:
Why this is empty?

Reply:
This error affects only the constructor function generation, therefore adding an output error message there or using a throwable did not make sense since it would either clutter the output or prevent generating the rest of the class. I will address this issue in a later PR by identifying all the incompletely generated bindings and giving a separate output or a detailed log file.